sentry-android-core/src/test/java/io/sentry/android/core/SystemEventsBreadcrumbsIntegrationTest.kt

```diff
@@ -2,25 +2,31 @@ package io.sentry.android.core
 
 import android.content.Context
 import android.content.Intent
+import android.os.Build
+import androidx.test.ext.junit.runners.AndroidJUnit4
 import io.sentry.Breadcrumb
 import io.sentry.IHub
 import io.sentry.ISentryExecutorService
 import io.sentry.SentryLevel
 import io.sentry.test.DeferredExecutorService
 import io.sentry.test.ImmediateExecutorService
+import org.junit.runner.RunWith
 import org.mockito.kotlin.any
 import org.mockito.kotlin.anyOrNull
 import org.mockito.kotlin.check
 import org.mockito.kotlin.mock
 import org.mockito.kotlin.never
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.whenever
+import org.robolectric.annotation.Config
 import kotlin.test.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
 import kotlin.test.assertNotNull
 import kotlin.test.assertNull
 
+@RunWith(AndroidJUnit4::class)
```

Comment:
hm, I think we are mocking out everything related to Android, why this needs to be a robolectric test?

Reply:
I was getting lint errors due to using the deprecated `.registerReceiver()` APIs without the e.g. `Context.RECEIVER_EXPORTED` flags.
I can also remove this and ignore the lint error instead.

Reply:
ah, it's alright then 👍